# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2022/03
# sanity check for guanaco fails on build nodes missing the nvidia driver.
# -> copy $EBROOTCUDA/lib64/stubs/libcuda* to %(installdir)s/lib
# remove the above when this file is executed in sanity check step.
diff -ru python-parakeet-0.2.7/src/parakeet/__init__.py python-parakeet-0.2.7_sanitycheck_fix/src/parakeet/__init__.py
--- python-parakeet-0.2.7/src/parakeet/__init__.py	2022-02-28 12:26:29.000000000 +0100
+++ python-parakeet-0.2.7_sanitycheck_fix/src/parakeet/__init__.py	2022-03-09 13:32:35.689067750 +0100
@@ -0,0 +1,4 @@
+print("Removing stubs libcuda* from $EBROOTPYTHONMINPARAKEET/lib") #SANITYCHECKFIX
+import os #SANITYCHECKFIX
+os.system("rm $EBROOTPYTHONMINPARAKEET/lib/libcuda* -f") #SANITYCHECKFIX
+os.system("sed -i '/SANITYCHECKFIX$/d' $EBROOTPYTHONMINPARAKEET/lib/python*/site-packages/parakeet/__init__.py" ) #SANITYCHECKFIX
